mmv1/products/notebooks/api.yaml

```diff
@@ -462,6 +466,480 @@ objects:
+              Currently supports one owner only.
+          - !ruby/object:Api::Type::String
+            name: 'proxyUri'
+            description: |
+              The proxy endpoint that is used to access the runtime.
+            output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: "softwareConfig"
+        description: |
+          The config settings for software inside the runtime.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: "notebookUpgradeSchedule"
+            description: |
+              Cron expression in UTC timezone for schedule instance auto upgrade.
+              Please follow the [cron format](https://en.wikipedia.org/wiki/Cron).
+          - !ruby/object:Api::Type::Boolean
+            name: "enableHealthMonitoring"
+            description: |
+              Verifies core internal services are running. Default: True.
+            default_value: true
+          - !ruby/object:Api::Type::Boolean
+            name: "idleShutdown"
+            description: |
+              Runtime will automatically shutdown after idle_shutdown_time.
+              Default: True
+            default_value: true
+          - !ruby/object:Api::Type::Integer
+            name: "idleShutdownTimeout"
+            description: |
+              Time in minutes to wait before shuting down runtime.
+              Default: 180 minutes
+          - !ruby/object:Api::Type::Boolean
+            name: "installGpuDriver"
+            description: |
+              Install Nvidia Driver automatically.
+          - !ruby/object:Api::Type::String
+            name: "customGpuDriverPath"
+            description: |
+              Specify a custom Cloud Storage path where the GPU driver is stored.
+              If not specified, we'll automatically choose from official GPU drivers.
+          - !ruby/object:Api::Type::String
+            name: "postStartupScript"
+            description: |
+              Path to a Bash script that automatically runs after a notebook instance
+              fully boots up. The path must be a URL or
+              Cloud Storage path (gs://path-to-file/file-name).
+      - !ruby/object:Api::Type::NestedObject
+        name: "metrics"
+        description: |
+          Contains Runtime daemon metrics such as Service status and JupyterLab
+          status
+        properties:
+          - !ruby/object:Api::Type::KeyValuePairs
+            name: "systemMetrics"
+            description: |
+              Contains runtime daemon metrics, such as OS and kernels and
+              sessions stats.
+            output: true
+        output: true
+      - !ruby/object:Api::Type::Time
+        name: 'createTime'
+        description: 'Runtime creation time'
+        output: true
+      - !ruby/object:Api::Type::Time
+        name: 'updateTime'
+        description: 'Runtime update time.'
+        output: true
```

Comment:
irrelevant to terraform... recommend removing

Reply:
Done